### PR TITLE
Added Docker setup in readme file

### PR DESCRIPTION
**Pull request by ankitpws**
Added Docker setup in readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,137 +1,245 @@
-# User Service APIs
+# User Service
 
-Recommend to,
-Install any IDE in your system(eg: VScode etc..)
-Install nodejs from : https://nodejs.org/en/download/
-Install mongoDB: https://docs.mongodb.com/manual/installation/
-Install Robo 3T: ​​https://robomongo.org/
+Elevate user services can be setup in local using three methods:
```
Comment (rakeshSgr) [inline README.md:3]: @ankitpws three methods or two methods?

File: README.md [hunk 2]
```diff
@@ -1,137 +1,245 @@
...
+    ELEVATE/user/src$ npm install
+    ```
+
+6. Start User server
+
+    ```
+    ELEVATE/user/src$ npm start
+    ```
+
+7. To set scheduler service job
```
Comment (rakeshSgr) [inline README.md:227]: @ankitpws there is no scheduler service job, remove this

File: README.md (hunk 1 above)
Comment (ankitpws) [reply README.md]: fixed

File: README.md [hunk 3]
```diff
@@ -1,137 +1,245 @@
...
 
-Recommend to,
-Install any IDE in your system(eg: VScode etc..)
-Install nodejs from : https://nodejs.org/en/download/
-Install mongoDB: https://docs.mongodb.com/manual/installation/
-Install Robo 3T: ​​https://robomongo.org/
+Elevate user services can be setup in local using three methods:
 
-## 1. Cloning the User repository into your system
+1. Dockerized service with local dependencies(Intermediate): Refer **Section A**.
```
[the remark this answers is not in the document]
Comment (ankitpws) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (ankitpws) [reply README.md]: fixed

**Pull request merged**